charts/coturn/values.yaml

```diff
@@ -26,9 +26,11 @@ coturnTurnListenPort: 3478
 coturnMetricsListenPort: 9641
 coturnTurnTlsListenPort: 5349
 
-# If you need to specify which IP Coturn should bind to.
-# This will typically be the IP of the kubenode.
-# coturnTurnListenIP: "182.168.22.133"
+# coturnTurnListenIP: "1.2.3.4" # can also be __COTURN_EXT_IP__, __COTURN_POD_IP__,__COTURN_HOST_IP__
```

Comment:
Some comment explaining the difference between `...ListenIp`, `...RelayIp` and `...ExternalIp` would be helpful.

Reply:
Wouldn't that just reproduce the [docs fo coturn](https://github.com/coturn/coturn/tree/master/docs)?

Reply:
Yes, but IMHO otherwise admins would need to understand `coturn` itself and not just look at our Helm charts. :thinking: